docker-compose-files/openshift-compose/README.md

```diff
@@ -0,0 +1,62 @@
+### OpenShift import docker-compose
+
+
+Start ADB/CDK
+
+To get latest openshift edit Vagrantfile make `IMAGE_TAG` to `IMAGE_TAG="latest"`
```

Comment:
I couldn't find the IMAGE_TAG in the CDK 2.0 Vagrantfile. Should it use another unreleased CDK version?